app/models/Product.coffee

```diff
@@ -4,3 +4,15 @@ module.exports = class ProductModel extends CocoModel
   @className: 'Product'
   @schema: require 'schemas/models/product.schema'
   urlRoot: '/db/products'
+
+  priceStringNoSymbol: -> (@get('amount') / 100).toFixed(2)
+
+  purchase: (token, options={}) ->
+    options.url = _.result(@, 'url') + '/purchase'
+    options.method = 'POST'
+    options.data ?= {}
+    options.data.token = token.id
+    options.data.timestamp = new Date().getTime()
+    options.data = JSON.stringify(options.data)
+    options.contentType = 'application/json'
+    return $.ajax(options)
```

Comment:
Now that we are more promise focused, I wonder if we should wrap these jQuery like promises in our real bluebird promises to avoid possible surprises down the road.